ote/src/cljs/ote/views/own_services.cljs

```diff
@@ -247,7 +247,8 @@
    [:ul.unstyled
     (doall
       (for [as oa-services]
-        [:li {:key (:service-id as)}
+        [:li {:key (:service-id as)
+              :data-cypress-op-id (:service-id as)}
```

Comment:
Wouldn't `:id` suite better?
Also, DOM doesn't have to refer to particular tool technology solution is implemented with so the `cypress` part is a bit striking naming wise. `:id`could be used by any functionality, not only e2e tests.

Reply:
https://docs.cypress.io/guides/references/best-practices.html according to cypress best practices using data attribute is the best because it's insulated from all changes.

Reply:
Great that this followed the tool convention, 
although I don't agree with this
"Your application may use dynamic classes or ID’s that change".  
When we're giging the id, we control if it changes or not. 
Semantically an "id" or "name" attribute sound more proper way on how to selecxt things than a "data". We just need to agree what convention we use :)

Reply:
I still think it's better to have data-attributes since with them, you can ben **100%** sure that they only affect tests. With id's you could in theory have css that targets that tag.